src/it/benchmark/pom.xml

```diff
@@ -14,7 +14,7 @@
     <packaging>jar</packaging>
     <properties>
         <jenkins.version>2.249</jenkins.version>
-        <java.level>8</java.level>
+        <java.level>11</java.level>
```

Comment:
It cannot be reverted while also accomplishing the purpose of this PR, which is stated in the PR description:

> Drop support for Java 8

The minimum Jenkins version cannot yet be updated because we do not know which version will drop support for Java 8, but the intention is that when we do we would update the minimum Jenkins version to that version.